task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -207,11 +207,16 @@ class XComOperations:
     def __init__(self, client: Client):
         self.client = client
 
-    def get(self, dag_id: str, run_id: str, task_id: str, key: str, map_index: int = -1) -> XComResponse:
+    def get(
+        self, dag_id: str, run_id: str, task_id: str, key: str, map_index: int | None = -1
```

Comment:
If this can be none, shouldn't that be the default?

Reply:
True, resolved in https://github.com/apache/airflow/pull/45075/commits/ae097e8a3cf26c2d3f9df6f97895a29a07a08500